authentication_lib/src/main/java/org/octri/authentication/server/security/entity/UserUserRole.java

```diff
@@ -17,7 +22,16 @@
  */
 @Entity
 @Table(uniqueConstraints = { @UniqueConstraint(columnNames = { "user_role", "user" }) })
-public class UserUserRole extends AbstractEntity {
+public class UserUserRole implements Serializable {
```

Comment:
This class cannot extend the common-lib `AbstractEntity`, because JPA auditing properties aren't supported by Hibernate's many-to-many join tables. The AuthLib `AbstractEntity` wasn't a problem, because it only held the ID attribute.